driver-sync/src/test/functional/com/mongodb/client/AbstractClientSideEncryptionTest.java

```diff
@@ -258,8 +258,12 @@ public void setUp() {
                         .addCommandListener(commandListener);
 
         if (!kmsProvidersMap.isEmpty()) {
-            String csflePath = System.getProperty("org.mongodb.test.csfle.path", "");
-            List<String> searchPath = csflePath.isEmpty() ? emptyList() : singletonList(csflePath);
+            String cryptSharedLibPath = System.getProperty("org.mongodb.test.crypt.shared.lib.path", "");
+            if (!cryptSharedLibPath.isEmpty()) {
+                extraOptions = new HashMap<>(extraOptions);
```

Comment:
Seems wrong to throw away any existing extraOptions by creating a new map here.
I'd be surprised if this doesn't cause a bunch of tests to fail.

Reply:
Oh, missed the copy.